ios-target/Model/Target.swift

```diff
@@ -0,0 +1,73 @@
+//
+//  Target.swift
+//  ios-target
+//
+//  Created by sol manini on 6/12/19.
+//  Copyright © 2019 TopTier labs. All rights reserved.
+//
+
+import Foundation
+
+class Target: Codable {
```

Comment:
why are you implementing the initializers? try using 
```swift
struct Target: Codable {
  var id: Int
  var title: String
  var lat: Double
  var lng: Double
  var radius: Double
  var topicId: Int
}
```
It should implement the decoding that you've done by default

Reply:
:exploding_head: